### PR TITLE
Readme modifications

### PR DESCRIPTION
**Pull request by ashyonline**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,39 @@
...
-It currently only works with video streamed from [libstreaming library](https://github.com/fyhertz/libstreaming) <br>
-running in a separate Android device. We are using a custom version of <br>
-libstreaming which is the original library plus some minimal changes made to <br>
-fix some issues we encountered while we were working on the Android Streaming <br>
-Client library.<br>
-
-##Content of the project
+It uses [efflux library](https://github.com/brunodecarvalho/efflux) to create an underlying RTP session to listen to <br>
+package arrival. <br>
+It only works with video streamed from [libstreaming library](https://github.com/fyhertz/libstreaming) running in a <br>
```
Comment (adamantivm) [inline README.md]: A small suggestion here: why not instead say that we have only tested so far with this client (as opposed to: it only works with this client).
I would also personally demote the "player" part of this to much later in the readme. I would make the readme much more about the library itself and how to use it, and put some information later about which players we have tested with and how.
This is only a suggestion.

**Comment by ashyonline** [comment]
@adamantivm what do you think now?

**Comment by adamantivm** [comment]
@ashyonline and I just talked about ideas for improvement:

 - Have someone else (not me) test this and make it work
 - I feel the part about libstreaming (publisher) is intermixed with the rest and doesn't need to
 - I would put the parts about "how to use it", specifically how to use this library to make a player, concise and all together
 -  I suggested thinking through how someone would use it, particularly thinking that someone who wants a player may already have a publisher (or would want a pointer to a publisher) and try to sketch the content accordingly (i.e.: in which ways can they connect their publisher to this player?)

**Comment by ashyonline** [comment]
@adamantivm I've made some changes. Can you merge it so I can go forward and I will then have someone else to test this and make it work?

Thanks

**Pull request merged**